yb-voyager/cmd/endMigrationCommand.go

```diff
@@ -250,22 +388,31 @@ func cleanupFallBackDB(msr *metadb.MigrationStatusRecord) {
 		return
 	}
 
-	utils.PrintAndLog("cleaning up fallback db...")
-	fbdb := tgtdb.NewTargetDB(msr.SourceDBAsTargetConf)
-	err := fbdb.Init()
+	utils.PrintAndLog("cleaning up voyager state from fallback db...")
+	var err error
+	fbconf := msr.SourceDBAsTargetConf
+	fbconf.Password = sourceDBPassword
+	if sourceDBPassword == "" {
+		fbconf.Password, err = askPassword("fallback DB", fbconf.User, "SOURCE_DB_PASSWORD")
```

Comment:
Why are we introducing the term "fall-back DB" when we can continue working with the term "source DB"??